SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Reporting Security Issues
+
+You can submit any security issue or suspected vulnerability
+on the GitHub Security pages of the project.
```

Comment:
I feel like I'm missing something here. I would expect `security@gradleup.com` here. Or is there a way to have some kind of "private issues" in GitHub for security?

Reply:
Turns out we can turn private issue reporting for vulnerabilites
https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/configuring-private-vulnerability-reporting-for-a-repository

Reply:
Nice! TIL.

Reply:
It is something we should add to the project onboarding checklist IMHO.
But yeah, it becomes a useful thing for small projects

Reply:
Raised https://github.com/GradleUp/gradleup.github.io/issues/2 for now